tasks/component-usage-graph.js

```diff
@@ -111,39 +115,42 @@ export function getDependantsLsStyle (componentPath, graph, maxLevel = Infinity)
 
 /**
  *
- * @param inputs
- * @param graph
- * @param mode
- * @param maxLevel
+ * @param {string[]} componentsPath
+ * @param {DepGraph} graph
+ * @param {'dependencies'|'dependants'} mode
```

Comment:
fix: mode is optional with default value:

```suggestion
 * @param {'dependencies'|'dependants'} [mode='dependencies']
```